src/main/webapp/i18n/de/programmingExercise.json

```
@@ -56,6 +56,7 @@
                 "steps": {
                     "generalInfoStepTitle": "Informationen",
                     "difficultyStepTitle": "Modus",
+                    "buildPlansTitle": "Build Pläne",
```

```suggestion
                    "buildPlansTitle": "Build-Pläne",
```
consistency with other translations